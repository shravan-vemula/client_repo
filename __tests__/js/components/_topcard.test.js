import React from 'react';
import {render,cleanup} from '@testing-library/react';
import TopCard from 'js/_components/molecules/_topcard';

afterEach(cleanup)

describe("TopCard Component",()=>{
    it("renders correctly", () => {
        const wrapper = shallow(<TopCard />);
        expect(wrapper).toMatchSnapshot();
      });

   
});