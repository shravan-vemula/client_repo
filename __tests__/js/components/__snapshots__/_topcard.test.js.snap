// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopCard Component renders correctly 1`] = `ShallowWrapper {}`;
